cvat/apps/dataset_manager/views.py

```diff
@@ -122,8 +122,8 @@ def export(dst_format, project_id=None, task_id=None, job_id=None, server_url=No
         instance_update_time = timezone.localtime(db_instance.updated_date)
         if isinstance(db_instance, Project):
             tasks_update = list(map(
-                lambda db_task: timezone.localtime(db_task.updated_date),
-                db_instance.tasks.all()
+                lambda updated_date: timezone.localtime(updated_date),
+                db_instance.tasks.values_list('updated_date', flat=True)
```

Comment:
It's even possible to request max directly from the DB. In either case, there will be new DB requests, as it's not `.all()`, which can be cached.